gulpfile.js

```diff
@@ -142,8 +142,10 @@ gulp.task('travis', ['lint', 'loader-polyfill', 'babel-app']);
  * Build the app.
  */
 gulp.task('build', function(cb) {
-  runSequence(['clobber-app'], ['loader-polyfill', 'copy-app'], ['babel-app'],
-    ['lint'], ['remove-useless'], cb);
+  runSequence(
+    ['lint'], ['clobber-app'], ['loader-polyfill', 'copy-app'], ['babel-app'],
```

Comment:
I believe linter should be the first one here to not waste time if something is wrong in the code